stanford_faq.module

```diff
@@ -7,11 +7,34 @@
 include_once 'stanford_faq.features.inc';
 
 /**
+ * Implements hook_preprocess_HOOK().
+ */
+function stanford_faq_preprocess_page(&$vars) {
+  $current_path = current_path();
```

Comment:
Why is this needed? arg(0) should work regardless of calling current_path(), no?